src/search-backend/default-search-backend.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { SearchParams } from '../search-params';
+import type { Result } from '@internetarchive/result-type';
+import type { SearchServiceError } from '../search-service-error';
+import { SearchParamURLGenerator } from '../search-param-url-generator';
+import { BaseSearchBackend } from './base-search-backend';
+import type { SearchBackendOptionsInterface } from './search-backend-options';
+
+/**
+ * The DefaultSearchBackend requests search results from archive.org without specifying a
+ * `service_backend`, causing the PPS to use the appropriate default for the current context.
+ */
+export class DefaultSearchBackend extends BaseSearchBackend {
+  private servicePath: string;
+
+  constructor(options?: SearchBackendOptionsInterface) {
+    super(options);
+    this.servicePath =
+      options?.servicePath ?? '/services/search/beta/page_production';
```

Comment:
QUESTION: Can we have @ximm also have `/services/search/page_production` accept requests with the same functionality, so that we can start using the latter URL in production?

Reply:
I would certainly be happy to point these at non-beta URLs once they're made available. I'll keep in mind that we should discuss that for the next migration steps.